programs/manifest/src/program/instruction_builders/deposit_instruction.rs

```diff
@@ -8,6 +8,7 @@ use solana_program::{
     pubkey::Pubkey,
 };
 
+// TODO: would be nice it it had a trader index hint as well
```

Comment:
Are we CU constrained? For code simplicity, didnt include optional hints on deposit and withdraw. I assumed that if you dont use global and dont leave enough funds on exchange, then you dont care about a few hundred CU

Reply:
ya left for todo bc. i didnt want to act on it, also ok if we later just get rid of it, if there's no time. in general: saving CU is good, so would still keep i open